events.html

```diff
@@ -24,8 +24,8 @@
     <section class="hero is-primary">
         <div class="hero-body">
             <div class="container has-text-centered">
-                <h1 class="title">
-                    <a href="./index.html">Primary title</a>
+                <h1 class="title">Primary title
```

Comment:
We still need this text to link to index.html, as users expect the title to load the page again